WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeErrorFragment.kt

```diff
@@ -0,0 +1,64 @@
+package org.wordpress.android.ui.publicize
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.os.Bundle
+import android.view.ContextThemeWrapper
+import androidx.annotation.StringRes
+import androidx.fragment.app.DialogFragment
+import org.wordpress.android.R
+import org.wordpress.android.R.style
+import org.wordpress.android.ui.WPWebViewActivity
+import org.wordpress.android.ui.publicize.PublicizeConstants.PUBLICIZE_FACEBOOK_SHARING_SUPPORT_LINK
+import java.lang.NullPointerException
+
+class PublicizeErrorFragment : DialogFragment() {
```

Comment:
Can we rename this one to `PublicizeErrorDialogFragment` for consistency with other dialog fragments?

Reply:
Sure! Thanks for spotting. Resolved a773842

Reply:
Perfect, thank you!